applications/asset_tracker_v2/doc/data_module.rst

```diff
@@ -96,6 +103,20 @@ CONFIG_DATA_ACCELEROMETER_THRESHOLD
 CONFIG_DATA_GNSS_TIMEOUT_SECONDS
    This configuration sets the GNSS timeout value.
 
+.. _CONFIG_DATA_SAMPLE_GNSS_DEFAULT:
+
+CONFIG_DATA_SAMPLE_GNSS_DEFAULT
+   This configuration includes GNSS during sampling.
+   Enabled by default.
+
+.. _CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT:
+
+CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT:
+	This configuration includes Neighbor cell measurements during sampling.
+   Enabled by default.
```

Comment:
```suggestion
.. _CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT:

CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT
   This configuration includes Neighbor cell measurements during sampling.
   Enabled by default.
```
This could fix the doc build issue.